plugins/vultr/api_key.go

```diff
@@ -0,0 +1,66 @@
+package vultr
+
+import (
+	"context"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func APIKey() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.APIKey,
+		DocsURL:       sdk.URL("https://www.vultr.com/api/#section/Authentication"),
```

Comment:
This link relates to the `vultr` API. From what I could see, the only documentation related to the CLI is within their repo's README (https://github.com/vultr/vultr-cli), and [this](https://www.vultr.com/news/How-to-Easily-Manage-Instances-with-Vultr-CLI/) article. Should we rather link to a section from one of these?

Reply:
I arrived at that link from the link as follows: CLI documentation on GitHub > Vultr API Key management page > Vultr API docs:

<img width="581" alt="Screenshot 2023-01-23 at 7 31 24 PM" src="https://user-images.githubusercontent.com/18581859/214207345-84a7286d-df29-4ee6-8b0f-42c1e54e84f8.png">

<img width="555" alt="Screenshot 2023-01-23 at 7 31 50 PM" src="https://user-images.githubusercontent.com/18581859/214207313-7368ae1e-f4b2-4fd4-b73b-7a7c30edddb3.png">

<img width="928" alt="Screenshot 2023-01-23 at 7 31 35 PM" src="https://user-images.githubusercontent.com/18581859/214207328-e369b9ea-1215-45e1-8cc2-15b0813a5c68.png">

Since `DocsURL` here refers about the credential itself and not about CLI docs, I am inclined to leave the current link as is. 🤔

Reply:
Thank you for expanding, Arun! 😄 That makes sense, I agree we should leave it like this.